File: sandbox/python_go/send.go
package main

import (
	"bufio"
	"fmt"
	"net"
	"os"
)

func main() {
	conn, err := net.Dial("udp", "127.0.0.1:16384")
	if err != nil {
		fmt.Println("Error", err)
		os.Exit(1)
	}
	defer conn.Close()

	fmt.Println("Enter the message")
	fmt.Println("Enter the 'exit' to terminate")

	buffer := bufio.NewScanner(os.Stdin)

	for {

		fmt.Print(">")

		if !buffer.Scan() {
			break
		}

		text := buffer.Text()

		if text == "exit" {
			break
		}

		conn.Write([]byte(text))
	}
}

File: sandbox/python_go/send.py
import socket

target_IP = "127.0.0.1"
target_PORT = 16384

sock = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)

print(f"Enter the message")
print("Enter the 'exit' to terminate")

while True:
    msg = input(">:")

    if msg == "exit":
        break

    sock.sendto(msg.encode('utf-8'), (target_IP, target_PORT))

sock.close()